packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -78,12 +78,15 @@ function EnableNotification({
       </div>
     );
   }
+  const message =
+    source === NotificationPromptSource.NewSource
+      ? 'Would you like to get notified on the status of your article submissions in real time?'
+      : `Want to get notified when ${
+          parentCommentAuthorName ?? 'someone'
+        } responds so you can continue the conversation?`;
```

Comment:
Prefer this to be a objectMap function if we can.
```
const message: <Type, string> = { 
NewSource: ''
Comment: ''
```

Reply:
Done 👍